archetypes/archetype-app-quickstart/src/main/resources/archetype-resources/pom.xml

```diff
@@ -16,7 +16,9 @@
         <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
         <aws.java.sdk.version>${javaSdkVersion}</aws.java.sdk.version>
         <slf4j.version>1.7.28</slf4j.version>
-        <graalvm.version>21.0.0</graalvm.version>
+        #if( $nativeImage == 'true')
```

Comment:
How is this parameter resolved?

Reply:
This is a parameter provided by users. https://github.com/aws/aws-sdk-java-v2/blob/master/archetypes/archetype-app-quickstart/src/main/resources/META-INF/maven/archetype-metadata.xml#L42-L43

Reply:
I thought it was, had a hard time finding it. Thanks!